src/app/public/modules/lookup/lookup.component.spec.ts

```diff
@@ -439,6 +490,53 @@ describe('Lookup component', function () {
 
         expect(lookupComponent.value).toEqual([]);
       }));
+
+      describe('form control interactions', function () {
+        it('should properly reset a form', fakeAsync(function () {
+          fixture.detectChanges();
+
+          performSearch('s', fixture);
+          selectSearchResult(0, fixture);
+
+          expect(getInputElement(lookupComponent).value).toBe('Isaac');
+          expect(lookupComponent.value).toEqual([{ name: 'Isaac' }]);
+
+          component.resetForm();
+
+          expect(getInputElement(lookupComponent).value).toBe('');
+          expect(lookupComponent.value).toEqual([]);
+        }));
+
+        it('should properly set a value through the form control', fakeAsync(function () {
+          fixture.detectChanges();
+
+          performSearch('s', fixture);
+          selectSearchResult(0, fixture);
+
+          expect(getInputElement(lookupComponent).value).toBe('Isaac');
+          expect(lookupComponent.value).toEqual([{ name: 'Isaac' }]);
+
+          component.setValue(0);
+
+          expect(getInputElement(lookupComponent).value).toBe('Andy');
+          expect(lookupComponent.value).toEqual([{ name: 'Andy' }]);
+        }));
+
+        it('should properly set a value through the form control', fakeAsync(function () {
```

Comment:
Again, the expectation should be distinct from the one above it.

Reply:
See my above comment.